src/components/Comment/Comment.js

```diff
@@ -68,28 +73,75 @@ const DeleteButton = styled.div`
   cursor: pointer;
 `;
 
-export default function Comment({ data, snippetId, userId, updateCommentList }) {
-  const { _id: commentId, creator, content, createdAt } = data;
+const InputArea = styled.input`
+  width: 850px;
+  height: 30px;
+  padding-left: 10px;
+  padding: 5px 10px 5px 13px;
+  border: none;
+  border-radius: 10px;
+  background-color: var(--caret-color);
+  color: white;
+  font-size: 20px;
+  outline: none;
+`;
+
+const SubmitIcon = styled.img`
+  width: 30px;
+  height: 25px;
+  margin-top: 1px;
+  margin-left: 20px;
+  cursor: pointer;
+`;
+
+export default function Comment({ commentData, snippetId, userId, updateCommentList }) {
+  const [isEditable, setIsEditable] = useState(false);
+  const [inputText, setInputText] = useState("");
+
+  const { _id: commentId, creator, content, createdAt } = commentData;
   const dateFormat = getDate(createdAt);
   const isCreator = creator._id === userId;
 
-  const handleOnclick = async () => {
-    const response = await deleteComment({ userId, commentId, snippetId });
+  const handleInputChange = (event) => {
+    setInputText(event.target.value);
+  };
```

Comment:
한 줄로 작성하는 방식으로 통일하면 어떨까요?😊

그리고 인자로 event 객체를 전달받을 때 구조 분해 할당을 통해 target 값만 받아오는 방법도 있습니다!

```suggestion
  const handleInputChange = ({ target }) => setInputText(target.value);
```